### PR TITLE
Jen Morse LiDAR Uncertainty .py and .ipynb files for review.

### PR DESCRIPTION
**Pull request by Jen-Morse**
# Pull Request Template

### Reproducibility
- [ ] Are the data downloaded in the code
- [ ] Are paths created to ensure they work on all operating systems (using  os.path.join)
- [ ] Are comments used to clarify the contents of the code that can’t be clarified using expressive variable and function names alone? (not too many comments - just enough)
- [ ] Does the notebook run from start to finish?

### Functions
- [ ] Do functions follow PEP 8 format conventions?
- [ ] Are function docstrings clear (all  inputs and outputs clearly  described and defined)
- [ ] Are function names expressive (the name describes what the function does)?
- [ ] Are functions easy to understand and read?
- [ ] How many tasks does each function do? (ideally a function does one thing well).

### Package imports
- [ ] Are standard modules (those included with the base  python install) vs. third party (related but externally developed tools) import groups correct with appropriate spacing in between each group?
- [ ] Are variable names throughout the code, expressive?
- [ ] Suggest changes if not, highlight what was done well
- [ ] Is the code overall easy to understand and read? Are there things that would make it more clear / cleaner?

### DRY Code
- [ ] Are segments of code repeated in the file or is the code DRY?
- [ ] Are loops used to optimize DRY code?
- [ ] Are functions used to optimize DRY code?
- [ ] Are there any areas that could be potentially improved (you can suggest improvements OR you can just highlight parts of the code where you suspect it could  be cleaner / more efficient.

### Novel Approaches to Problem solving
- [ ] Highlight any novel approaches to completing the assignment.

### CODE REVIEW

**Comment by eculler** [comment]
@allisonw5 Please review this PR #56  (Canvas isn't working so let's try GitHub directly!)

File: Jen-Morse-lidar-uncertainty.py
Comment (allisonw5) [inline Jen-Morse-lidar-uncertainty.py:1]: @Jen-Morse looks like your data is downloaded and paths are in place! Great!

Comment (allisonw5) [inline Jen-Morse-lidar-uncertainty.py:1]: @Jen-Morse This Looks great! Very fancy class. While I don't exactly grasp them yet, I like the use of the property decorators. It is all well formatted and easy to read- the code, docstrings, and the comments. I like the plotting function too. I like it more than mine. Well done!

Comment (Jen-Morse) [reply Jen-Morse-lidar-uncertainty.py:1]: Thanks @allisonw5!

Comment (Jen-Morse) [reply Jen-Morse-lidar-uncertainty.py:1]: Thanks @allisonw5 , I feel like there were still a few things to work out in my plotting function! Be in touch if you want to work on those functions together at all. :)

**Comment by eculler** [comment]
nice work, Jen! Great use of classes and functions. Next step - make sure to include parameters and returns and attributes in your docstrings so others can know what your code does.

**Pull request closed without merging**